dev/core/src/com/google/gwt/dev/jjs/impl/gflow/cfg/CfgBuilder.java

```diff
@@ -393,16 +377,14 @@ public boolean visit(JBreakStatement x, Context ctx) {
     @Override
     public boolean visit(JCaseStatement x, Context ctx) {
       pushNode(new CfgStatementNode<JStatement>(parent, x));
-      if (x.getExpr() != null) {
+      if (x.isDefault()) {
+        // default label
+      } else {
```

Comment:
Should we invert this if and remove the empty block?

Reply:
Yes, I will do that.